### PR TITLE
ProfilePage

### PR DESCRIPTION
**Pull request by mrana17**
✨ Create ProfilePage

- Header with Header Buttons
- Subheader
- Choose Ride Button
-Calendar
-Footer with FooterButtons

- add CSS

- change global.css to prevent overflow

### CODE REVIEW

File: pages/ProfilePage.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import SubHeader from "../components/subHeader/subHeader";
+import styles from "../styles/ProfilePage.module.css";
+import Calendar from "react-calendar";
+import "react-calendar/dist/Calendar.css";
+import FooterButton from "../components/footerButton/footerButton";
+
+export default function LoginPage() {
+  return (
+    <div className={styles.container}>
+      <Head>
```
Comment (RonjaMeinecke) [inline pages/ProfilePage.tsx]: you can put this header in the  _app.tsx so you don't have to import in on every page

File: pages/ProfilePage.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <title>Login Page</title>
+        <link rel="icon" href="/full-Logo.svg" />
+      </Head>
+
+      <main className={styles.main}>
```
Comment (RonjaMeinecke) [inline pages/ProfilePage.tsx]:
```suggestion
      <main className={styles.container}>
```

**Review by RonjaMeinecke — APPROVED** [review]
🦾GJ

File: pages/LoginPage.tsx [hunk 3]
```diff
@@ -18,10 +19,7 @@ export default function LoginPage() {
           <Input type="text" placeholder="Benutzername" />
           <Input type="password" placeholder="Passwort" />
         </div>
-        <Button
-          label="Login"
-          onClick={() => alert("next page is coming soon")}
-        />
+        <Button label="Login" onClick={() => router.push("/ProfilePage")} />
```
Comment (m-wiemers) [inline pages/LoginPage.tsx:16]: I would write filenames and routes in small letters

```suggestion
        <Button label="Login" onClick={() => router.push("/profilepage")} />
```

**Review by m-wiemers — APPROVED** [review]
gj 🚀

**Pull request merged**